src/Components/Category/Category.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'react-bootstrap/Image';
+
+import './Category.scss';
+import '../../Styles/fontello/css/fontello.css';
+
+function Category({ name, icon }) {
+  return (
+      <div className="icon-div" xs={12} md={6} lg={3} xl={3} >
```

Comment:
Nie wrzucaj w nazwę klasy nazwy tagu, w której ta klasa się znajduje. Poza tym nazwa tej klasy jest nieadekwatna do tego, co próbujesz tutaj zrobić, przecież to nie jest ikona, tylko kategoria - więc klasa raczej powinna się nazywać po prostu `category`.